### ISSUE TITLE
Improve documentation for ParticleProjectionPlot weight_field

### ISSUE THREAD

**Issue by osterwood** [yt-project/yt#1608]
### Bug report

**Bug summary**

The default of `weight_none` is not appropriate for many kinds of fields that ParticleProjectionPlot can plot.  This affects the color mapping and is not currently well documented.

The addition of `weight_field='particle_ones'` to the code creates the expected output.  The documentation for [ParticleProjectionPlot](http://yt-project.org/doc/reference/api/yt.visualization.particle_plots.html?highlight=particleprojectionplot#yt.visualization.particle_plots.ParticleProjectionPlot) mentions the weight field, but does not provide any examples of its use or list valid options for it. 

**Code for reproduction**

```python
#!/usr/bin/env python

import os, sys, inspect, time
import math, string

import numpy as np
import yt

CMAP_NAME = 'discrete'
COUNT     = 100000
OFFSET    = 8

yt.make_colormap([('blue', 20), ('green', 20), ('red', 20)],
                 name=CMAP_NAME, interpolate=False)

zeros = np.array([np.zeros(COUNT)])

# Create three clusters of X,Y,Z,Temp data
# Each cluster has a different XYZ position and a different (fixed) temperature
A = np.concatenate((np.random.randn(COUNT,3), zeros.T), axis=1) + [0,0,OFFSET, 1]
B = np.concatenate((np.random.randn(COUNT,3), zeros.T), axis=1) + [0,OFFSET,0, 2]
C = np.concatenate((np.random.randn(COUNT,3), zeros.T), axis=1) + [OFFSET,0,0, 3]

data = np.concatenate((A, B, C), axis=0)

print np.shape(data)

Xs = data[:,0]
Ys = data[:,1]
Zs = data[:,2]
Is = data[:,3]

print "" 
print "Compute Bounding Box" 

PADD = 1.0

min_x = np.min(Xs) - PADD
min_y = np.min(Ys) - PADD
min_z = np.min(Zs) - PADD

max_x = np.max(Xs) + PADD
max_y = np.max(Ys) + PADD
max_z = np.max(Zs) + PADD

params = {
    'cmap'    : CMAP_NAME,
    'field'   : 'temperature'
}

data = {'particle_position_x': Xs,
        'particle_position_y': Ys,
        'particle_position_z': Zs,
        params['field']: Is}

print "Rendering"

bbox = np.array([[min_x, max_x], [min_y, max_y], [min_z, max_z]])

ds = yt.load_particles(data, length_unit="ft", mass_unit='kg', bbox=bbox)

p  = yt.ParticleProjectionPlot(ds, 1, params['field']) 
p.set_cmap(field=params['field'], cmap=params['cmap'])
p.set_background_color(params['field'], color='white')
p.save('example_weight_none.png') 

p  = yt.ParticleProjectionPlot(ds, 1, params['field'], weight_field='particle_ones') 
p.set_cmap(field=params['field'], cmap=params['cmap'])
p.set_background_color(params['field'], color='white')
p.save('example_weight_one.png') 
```

**Actual outcome**

![example_weight_none](https://user-images.githubusercontent.com/10534/32394321-1bd33a4c-c0b3-11e7-8153-3b35080fa04d.png)

**Expected outcome**

![example_weight_one](https://user-images.githubusercontent.com/10534/32394329-212ee158-c0b3-11e7-905b-8f39e67b6dcb.png)

**Comment by ngoldbaum** [comment]
Thanks for the report and welcome to the yt community! 🎆 💯

**Comment by ngoldbaum** [comment]
@munkm volunteered to look at this

**Comment by cphyc** [comment]
Is this still relevant?

**Comment by ngoldbaum** [comment]
Fixed by #2058
